artemis-pom/pom.xml

```diff
@@ -524,10 +524,10 @@
             <!-- License: (Joint): Apache 2.0 or EPL 2.0 -->
          </dependency>
          <dependency>
-            <groupId>org.eclipse.jetty.toolchain</groupId>
-            <artifactId>jetty-servlet-api</artifactId>
-            <version>${jetty-servlet-api.version}</version>
-            <!-- License: (Joint): Apache 2.0 or EPL 2.0 -->
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
+            <version>${jakarta.servlet-api.version}</version>
+            <!-- License: EPL 2.0 -->
```

Comment:
Related to earlier comment, think it should be left at the jetty one (now jetty-jakarta-servlet-api to match the newer jetty).